json-resume-builder/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+import resumeData from "./resume.json";
+
+import Header from "./components/Header";
+import Education from "./components/Education";
+import Summary from "./components/Summary";
+import Projects from "./components/Projects";
+import Work from "./components/Work";
+
```

Comment:
It looks like you are importing your resume data into each component. You can look into passing props to each component. 

It's pretty easy to set up 

it'll be <Education varName={Vardata} />

then education will have access to your data through props.

Reply:
Yah i thought about that afterwards, but was getting too low on time and just had to keep powering through lol